### DISCUSSION TITLE
[5.x] Are entry type handles in unique in Craft 5?

### DISCUSSION THREAD

**Discussion by janhenckens** [craftcms/cms#14094] [Q&A] [answered]
I'm upgrading a plugin that already stored json data in Craft 4 and I'm trying to work around getting `getValueSql()` in my QueryBehaviour. 

I can have to user pass the entryType handle as those appear to be unique (creating a new type with an existing handle results in a validation error).

However, the function to get the entry type by handle is plural: `getEntryTypesByHandle`.

This that a left over function name from Craft 4 or am I missing something? Thanks!

**Comment by brandonkelly** [comment, marked as answer]
That method was overlooked as it’s hardly used within Craft, but already fixed it for the next release (061b86369588b07eac3d051b6390b50c97cf31e7) 😄

**Reply by brandonkelly** [reply]
Craft 5.0.0-alpha.4 is tagged now with that change. So now it’s just `getEntryTypeByHandle()`.

**Reply by janhenckens** [reply]
Thanks!
